pkg/config/config.go

```diff
@@ -45,48 +46,22 @@ func GetProviderFromInfrastructure(infra *configv1.Infrastructure) (configv1.Pla
 
 // getImagesFromJSONFile is used in operator to read the content of mounted ConfigMap
 // containing images for substitution in templates
-func getImagesFromJSONFile(filePath string) (imagesReference, error) {
+func getImagesFromJSONFile(filePath string) (ImagesReference, error) {
 	data, err := ioutil.ReadFile(filepath.Clean(filePath))
 	if err != nil {
-		return imagesReference{}, err
+		return ImagesReference{}, err
 	}
 
-	i := imagesReference{}
+	i := ImagesReference{}
 	if err := json.Unmarshal(data, &i); err != nil {
-		return imagesReference{}, err
-	}
-	if i.CloudControllerManagerOperator == "" {
-		return imagesReference{}, fmt.Errorf("operator image was not found in images ConfigMap")
+		return ImagesReference{}, err
 	}
 	return i, nil
 }
 
-// getCloudControllerManagerFromImages returns a CCM binary image later used in substitution
-func getCloudControllerManagerFromImages(platform configv1.PlatformType, images imagesReference) string {
-	switch platform {
-	case configv1.AWSPlatformType:
-		return images.CloudControllerManagerAWS
-	case configv1.OpenStackPlatformType:
-		return images.CloudControllerManagerOpenStack
-	case configv1.AzurePlatformType:
-		return images.CloudControllerManagerAzure
-	default:
-		return ""
-	}
-}
-
-func getCloudNodeManagerFromImages(platform configv1.PlatformType, images imagesReference) string {
-	switch platform {
-	case configv1.AzurePlatformType:
-		return images.CloudNodeManagerAzure
-	default:
-		return ""
-	}
-}
-
 // ComposeConfig creates a Config for operator
 func ComposeConfig(infrastructure *configv1.Infrastructure, clusterProxy *configv1.Proxy, imagesFile, managedNamespace string) (OperatorConfig, error) {
-	platform, err := GetProviderFromInfrastructure(infrastructure)
```

Comment:
From chat conversation with @Fedosin: need to rewrite this function, in order to make it's purpose more clean. Currently it serves for `infrastructure` object validation purposes before further usage.